generate.js

```diff
@@ -12,11 +12,25 @@ ncp('static', 'out/static', err => {
   }
 })
 
+let i = orgs.length
+let noLeader = []
+while (i--) {
+  if (!orgs[i].leaders.length) {
+    noLeader.push(orgs[i])
+    orgs.splice(i, 1)
+  }
+}
```

Comment:
You could use `.filter`
`.push` can be done inside the callback.

Reply:
I try to do this
```es6
const noLeader = orgs.filter((org, index) => {
  if (!org.leaders.length){
    orgs.splice(index, 1)
    return true
  }
  return false
})
```
but if i splice orgs array, ..., should I make temp array? any suggestion

Reply:
if i use filter and splice orgs, it will not working

Reply:
But it's returning a new array so you need to do 
```js
orgs = orgs.filter(..
```